### PR TITLE
kraken: update car edges cost with car speed if available

### PR DESCRIPTION
**Pull request by kinnou02**
follow up of #3088 
read only https://github.com/CanalTP/navitia/commit/b2a4f6c3650c88e08e7eaa2aabef4a73e51b264d

### CODE REVIEW

**Review by eturck — APPROVED** [review]

File: source/ed/connectors/speed_parser.cpp [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+#include <iostream>
+#include <algorithm>
+#include "utils/exception.h"
+#include <boost/algorithm/string/trim.hpp>
+#include <boost/optional/optional_io.hpp>
+
+namespace ed {
+namespace connectors {
+
+constexpr float kmh_to_ms(float val) {
```
Comment (xlqian) [inline source/ed/connectors/speed_parser.cpp:43]: `constexpr long double operator"" _kmh (double val )` ?

Comment (eturck) [reply source/ed/connectors/speed_parser.cpp:43]: This code was already pushed in #3088 maybe it should be there?

**Comment by kinnou02** [comment]
How do you want to proceed on those PR? I won't be able to work on it for a while at least.

**Review by AurelienLP — APPROVED** [review]

**Review by xlqian — APPROVED** [review]

**Review by benoit-bst — APPROVED** [review]

**Comment by kinnou02** [comment]
:tada:

**Pull request merged**